src/main/resources/static/api.yml

```diff
@@ -0,0 +1,616 @@
+openapi: 3.0.0
+info:
+  title: Grid Dynamics Internship -  Hospital Management
+  description: |-
+    A simple application demo for hospital management created at the Grid Dynamics Java internship. The app uses Java 17, Spring boot, Gradle, Hibernate, Postgres + database migration tool (Liquibase), Lombok, Swagger.
+  version: 0.0.1
+servers:
+  - url: https://localhost:8080
+tags:
+  - name: Doctors
+    description: Available commands for doctors
+  - name: Nurses
+    description: Available commands for nurses
+  - name: Admin
+    description: Available commands for admin / auth
+paths:
+  /treatments:
```

Comment:
we don't need this endpoint

Reply:
Wasn't sure about this... shouldn't the doctors be able to create and modify a catalogue of medicines/procedures? This because the prescriptions entity uses a `treatmentId` to specify which treatment is associated with the prescription. Added a CRUD tag/controller for these endpoints this commit, will delete if you think it should be handled differently